essays/AITBW-roadToDune/index.md

```diff
@@ -0,0 +1,83 @@
+
+Two and a half years into college, I faced somehow of a career crisis (which is way more common that you would think) and questioned myself if I had chosen the right path for me. One of my closest, dearest friends told me during that time that _we weren't supposed to become programmers; we're studying Engineering to become Engineers, not workers on a construction site_ and in someway he was right, but something inside me told me I needed to understand programming if I wanted to become a great Engineer.
+
+That was around mid 2012 and nothing really relevant happened after that until late 2014 where I got interested into programming again —decided to get it right this time and began researching about (programming) languages others than the ones I had learned in college.
+
+Besides C# and PHP, I learned Java. (Along with Classic ASP (yep) and JSP). I despised all 3 of them.
+
+* PHP is an abomination from hell. My thoughts about it are all backed on my own experiences and this incredible [article](https://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/)
+
+* Java is extremely verbose, bloated and Oracle is not the most FOSS-friendly enterprises of them all. (This "hate" made me aware of Scala which I find beautiful and expressive without all the bad stuff Java has; I plan to get into it in the future)
+
+[scala_vs_java](./images/scala_vs_java.png)
+
+* C# is the better of this pack and some pretty cool stuff comes with it but I just can't stand Windows for development —Don't get me wrong: I use Windows a lot for gaming, photography and audio-related stuff but for development is just "meh" (WSL wasn't even close back then, not even in our [wildest dreams](https://www.youtube.com/watch?v=IdneKLhsWOQ))
+
+So after spending a few weeks researching about programming languages I wasn't aware of (Haskell and Clojure were the highlights of that research), I found Ruby and I immediately fell in love because of its syntax and how easy it was to get stuff done in a few lines of code and I couldn't wait to get into a project where my new found knowledge could be used.
+
+Right before New Years' Eve (2 days before, if I recall correctly), I landed an internship (a requirement to opt for the Engineering degree) that began on January 2015 from April 2015, as a _Backend Developer_ on a government-backed company, here on my hometown.
+
+I was assigned, along with a friend, the task to develop a web-based system for the company. «Sure thing», I said and also proposed «Can we do this on Ruby? I just found about it and seemed like a great choice for this project», but the manager asked for it on PHP since they had their stack on PHP.
+
+I wasn't too happy about it but I needed the internship —plus, I could work from home.
+
+> What I thought was going to be a terrible experience, happen to became a turning point in my career as a developer
```

Comment:
Same feedback about the quotation syntax.

Reply:
See [here](https://github.com/tecnoyucas/Publication-0/pull/4#discussion_r168975755)